app/src/main/res/layout-sw600dp-land/resume_lesson_fragment.xml

```diff
@@ -48,7 +48,7 @@
       android:layout_width="0dp"
       android:layout_height="match_parent"
```

Comment:
The constraintLayout background needs to be updated.

For now you can keep light and dark mode color same but make sure that you mention that as an issue in "Post PR changes" in our document.

Reply:
Done.